shell/agents/AIShell.Azure.Agent/AzCLI/AzCLIChatService.cs

```diff
@@ -10,7 +10,7 @@ namespace AIShell.Azure.CLI;
 
 internal class AzCLIChatService : IDisposable
 {
-    internal const string Endpoint = "https://azclitools-copilot-apim-temp.azure-api.net/azcli/copilot";
+    internal const string Endpoint = "https://azclitools-copilot-apim-test.azure-api.net/azcli/copilot";
```

Comment:
@wyunchi-ms @jsntcy I have been using the "temp" endpoint because it seems to be the dev branch/deployment where Xing Zhou has been fixing issues that I reported. I don't know Xing's GitHub account, can you please ask Xing to confirm which one should be used?

This is quoted from Xing's comment on Teams:
> Recently, we updated the environment for parameter value injection project and dogfood
parameter value injection env (temp env):
https://azclitools-copilot-apim-temp.azure-api.net/azcli/copilot
dogfood env (test env):
https://azclitools-copilot-apim-test.azure-api.net/azcli/copilot
Could you please help redirect the request of client side to these two URLs?

Reply:
@wyunchi-ms According to Xing, we should keep using the temp env endpoint because this branch contains all the parameter injection work. So, please revert this change.